cdap-spark-core-base/src/main/java/co/cask/cdap/app/runtime/spark/classloader/SparkClassRewriter.java

```diff
@@ -597,57 +604,84 @@ public MethodVisitor visitMethod(int access, String name, String desc, String si
         }
 
         // Wrap the original main method with
-        // Cancellable cancel = SparkRuntimeUtils.initSparkMain();
+        // CompletableFuture<?> completion = SparkRuntimeUtils.initSparkMain();
```

Comment:
this and other comments are inconsistent with the actual API

Reply:
fixed